cognite/neat/session/_session/_data_model/_routes.py

```diff
@@ -43,3 +58,103 @@ def _repr_html_(self) -> str:
             output.append(f"<H2>Data Model</H2><br />{html}")
 
         return "<br />".join(output)
+
+    def convert(self, reserved_properties: Literal["error", "warning"] = "warning") -> IssueList:
```

Comment:
Should we call this `to_physical`? Convert was used for both directions before, but now it is only used in one way.